app/scripts/controllers/ts/BaseController.js

```diff
@@ -0,0 +1,78 @@
+(function (factory) {
```

Comment:
This file built would normally be ignored. It was included in the PR so the gulp build pipeline or CI didn't need to be reconfigured for this POC.